### ISSUE TITLE
Localhost now is a required argument

### ISSUE THREAD

**Issue by EmbeddedMike** [node-inspector/node-inspector#604]
I've just upgraded to Node Inspector v0.9.2 
My command line is:

`node-debug --no-preload server.js`

This used to work. I now get this message:

`Malformed URL
http:/debug?port=5858`

I now have to use

`node-debug --web-host localhost --no-preload server.js`

**Comment by bajtos** [comment]
I am not able to reproduce the problem using the latest Node Inspector version 0.10.0 running on Node 0.10.38.

```
 node-debug --no-preload simple-server/
Node Inspector is now available from http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858
Debugging `simple-server/`

debugger listening on port 5858
```

Is there anything special about your environment? What is your operating system?

**Comment by EmbeddedMike** [comment]
OK I have an answer for this.
The regression is in the handling of the .nodeinspectorrc

My old version:
```
{
  "web-port": 8088,
  "web-host": null,
  "debug-port": 5858,
  "save-live-edit": true,
  "preload": false,
  "hidden": [],
  "stack-trace-limit": 50
}
```
It used to work, but does not with 0.10

However, if I remove this line

```
  "web-host": null //REMOVE
```
Or if I change it to:
```
  "web-host": "localhost"
```
Then it works.
My env
```
$ node-debug -v
Node Inspector v0.10.0

$ node -v
v0.12.2

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 13.10
Release:        13.10
Codename:       saucy
```

**Comment by bajtos** [comment]
Cool, thanks for investigating the issue. Could you submit a patch to fix the issue please? The configuration is loaded by [lib/config.js](https://github.com/node-inspector/node-inspector/blob/master/lib/config.js).

**Comment by EmbeddedMike** [comment]
I would be happy to do this but am on a trip and will not have access for a
month.

**Comment by jcrben** [comment]
Even if I run node-debug --webhost localhost [myapp], I still have to access my app served at 127.0.0.1:[port] rather than localhost:[port]. Is that what this is getting at or should I open a new issue?

Meanwhile, running my app with node normally allows it to be accessed with localhost or 127.0.0.1 interchangeably.

**Comment by bajtos** [comment]
I think the problem may be caused by using IPv4-only `127.0.0.1` as a default hostname([source code](https://github.com/node-inspector/node-inspector/blob/21dee3d0cd33d9a08ce5057e4a89dfc0b3756abb/lib/config.js#L439). `localhost` is resolved as an IPv6 address, where we don't listen on. That does not explain why `--webhost localhost` does not work though.

**Comment by 3y3** [comment]
Will be fixed in 0.11.0

**Comment by timbl** [comment]
Maybe this needs a new issue, maybe the same.  It is IPV6 confusion, 'localhost' can resolve to either IPV4 or IPV6  ... so I get 
```
$ node --version
v6.4.0
$ node  --debug-brk  foo.js
Debugger listening on [::]:5858
```
Where the [::] looks like the v6 equivalent of 128.0.0.1  .  So in this case, node inspector never connects to it.    Node itself doesn't allow --web-host as an option it seems, only node-inspector.
node-inspector --version v0.7.4
